pulp_ansible/app/viewsets.py

```diff
@@ -327,6 +329,30 @@ def sync(self, request, pk):
         )
         return OperationPostponedResponse(result, request)
 
+    @extend_schema(
+        description="Trigger an asynchronous task to rebuild Ansible content meta.",
+        responses={202: AsyncOperationResponseSerializer},
+    )
+    @action(detail=True, methods=["post"], serializer_class=AnsibleRepositoryRebuildSerializer)
```

Comment:
Defining an empty `inline_serializer` here should do the trick for drf-spectacular.

https://drf-spectacular.readthedocs.io/en/latest/drf_spectacular.html?highlight=inline_serializer#drf_spectacular.utils.inline_serializer